src/liballoc/collections/linked_list.rs

```diff
@@ -3,11 +3,11 @@
 //! The `LinkedList` allows pushing and popping elements at either end
 //! in constant time.
 //!
-//! Almost always it is better to use `Vec` or [`VecDeque`] instead of
-//! [`LinkedList`]. In general, array-based containers are faster,
-//! more memory efficient and make better use of CPU cache.
+//! NOTE: It is almost always better to use [`Vec`] or [`VecDeque`]...
```

Comment:
Why the `...` at the end now?

Reply:
A period (`.`) feels abrupt. The `...` indicates a gentle connection between the before (claim) and after (justification).

Also, am sure there's a better explanation why people use `...`, which is something of a subtle concept.

Reply:
I often use "..." in casual conversation because it suggests that I'm not done thinking; it's a sort of "uhm", "eh". but in a more formal setting (i.e. standards documentation) it just feels sloppy. See also https://en.wikipedia.org/wiki/Speech_disfluency and https://en.wikipedia.org/wiki/Filler_(linguistics).

Reply:
Alternative:

> NOTE: It is almost always better to use `Vec` or `VecDeque` because array-based containers are generally faster...

As a sidenote, and assuming you like that better, do you think we should add a comma there...

> NOTE: It is almost always better to use `Vec` or `VecDeque`, because array-based containers are generally faster...

Reply:
The first alternative is fine.

Reply:
is done